### PR TITLE
Allow specifying optional application namespace property in Third Party systems registration

### PR DESCRIPTION
**Pull request by emmakarova**
**Description**

Added Optional field for the placeholders when registering application from template

Changes proposed in this pull request:
- ...

**Related issue(s)**

- ...

**Pull Request status**

- [ ] Implementation
- [ ] Unit tests
- [ ] Integration tests
- [ ] `chart/compass/values.yaml` is updated 
- [ ] Mocks are regenerated, using the automated script

### CODE REVIEW

File: chart/compass/values.yaml [hunk 1]
```diff
@@ -139,7 +139,7 @@ global:
       name: compass-pairing-adapter
     director:
       dir:
-      version: "PR-2898"
```
Comment (dzahariev) [inline chart/compass/values.yaml:142]: Bump also tests version

**Review by dzahariev — CHANGES_REQUESTED** [review]

**Comment by emmakarova** [comment]
Replaced by https://github.com/kyma-incubator/compass/pull/2907

**Pull request closed without merging**